src/.fantomasignore

```diff
@@ -0,0 +1 @@
+TaskSeqInternal.fs
```

Comment:
if you move sln up, can this redundancy be removed?

Reply:
Yeah, I have thought about that. There was a reason why I put the `sln` in the `src` folder, but I can't readily recall. If I change it, and I probably should, I would do that in a separate PR to not pollute this one.

The `.fantomasignore` is going to be temporary anyway.

Reply:
Generally the reason I run into is that if you have a `.proj` and a `.sln` in the same folder, dotnet build and things start complaining about ambiguity
Agree separate PR makes sense for that

Reply:
yeah, but that's not the case here. There is no `proj` and `sln` in the same folder. It is just one down from the root, but structure is standard.

Having them an extra directory apart gives issues in Visual Studio when you add a new project, which is will add by default to the root dir (in a project dir), but not under `src`. Which is probably why i did it this way.

Neither approach would solve the issue with Fantomas, btw.

Reply:
>Neither approach would solve the issue with Fantomas, btw.

Then don't do that :D

But yes, having a separate `src` vs `tests` split is obv only actually useful once you stop having 1:1 sets of X and X.Tests and/or something that you'd put in a Directory.props file which would vary between tests and src (and/or enable you not to put redundant stuff at top level)

>Neither approach would solve the issue with Fantomas, btw.

pity, and I guess thats the main concern

Reply:
Well, good news, in part, as it was suggested to use Fantomas alpha release (see https://github.com/fsprojects/fantomas/issues/3019#issuecomment-1863079619) to at least get rid of the `.fantomasignore` file again.

We still have the double manifest, but at least we can now format `WhileBang` properly.

Reply:
I reverted that change and rebased.